packages/action/index.js

```diff
@@ -22,32 +22,78 @@ const core = require('@actions/core');
 async function main() {
   try {
     // `who-to-greet` input defined in action metadata file
+    if(!core.getInput('repo-token') || !core.getInput('os') || !core.getInput('file-path') || !core.getInput('repo-token') ){
+      throw "Github action missing required fields. Refer to documentation at https://github.com/googlecloudplatform/flaky-service";
```

Comment:
Can be a separate PR, but it doesn't look like the root README has relevant docs.